### PR TITLE
redundancy attributes logic removed

### PR DESCRIPTION
**Pull request by komushi**
# Description
To remove redundant attributes in target.dao of notification service

## Type of change

- [*] *Bug fix (non-breaking change which fixes an issue)*
- [ ] *New feature (non-breaking change which adds functionality)*
- [ ] *Breaking change (fix or feature that would cause existing functionality to not work as expected)*
- [ ] *Refactor* (existing code being refactored)
- [ ] *This change requires a documentation update*

# Submission Checklist

- [*] Build Verified
- [*] Bundle Verified
- [*] Lint passing
- [*] Unit tests passing
- [*] Integration tests passing
- [ ] Change logs generated

### CODE REVIEW

**Review by hassankhokhar — CHANGES_REQUESTED** [review]
No commented code in production. Also, is this verified by enhancing unit tests or integration tests ? I see we removed a bunch of code with no changes to unit or integration tests. How do we know this is backwards compatible or we haven't broken any functionality ?

**Comment by boardthatpowder** [comment]
Please generate change logs (run `rush change`) and add to the PR. Thanks.

File: source/packages/services/events-processor/src/api/targets/target.dao.spec.ts [hunk 1]
```diff
@@ -63,10 +63,7 @@ describe('TargetDao', () => {
                 pk: `S:${stubbedGoodEmailItem.subscriptionId}`,
                 sk: `ST:email:${stubbedGoodEmailItem.address}`,
                 gsi2Key: `ES:${eventSourceId}:${principal}:${principalValue}`,
-                gsi2Sort: `S:${stubbedGoodEmailItem.subscriptionId}:ST:email:${stubbedGoodEmailItem.address}`,
```
Comment (hassankhokhar) [inline source/packages/services/events-processor/src/api/targets/target.dao.spec.ts:66]: Can you provide explanation on why are these removed form the unit tests ?

File: source/packages/services/events-processor/src/api/targets/target.dao.ts [hunk 2]
```diff
@@ -43,12 +43,13 @@ export class TargetDao {
             gsi2Sort: createDelimitedAttribute(PkType.Subscription, item.subscriptionId, PkType.SubscriptionTarget, item.targetType, item.getId())
         };
 
-        for (const prop of Object.keys(item)) {
-            const value = item[prop];
-            if (value !== undefined) {
-                putItemAttributeMap[prop]= item[prop];
-            }
-        }
+        // Redundancy as the attributes added here are never to be used anywahere
```
Comment (hassankhokhar) [inline source/packages/services/events-processor/src/api/targets/target.dao.ts:46]: no commented  code in production.

**Review by hassankhokhar — CHANGES_REQUESTED** [review]

File: source/packages/services/events-processor/src/api/targets/target.dao.spec.ts (hunk 1 above)
Comment (hassankhokhar) [reply source/packages/services/events-processor/src/api/targets/target.dao.spec.ts:66]: @tsugunao still need your input on the question

Comment (tsugunao) [reply source/packages/services/events-processor/src/api/targets/target.dao.spec.ts:66]: @hassankhokhar I am not get involved with this change. I guess you meant @komushi

Comment (komushi) [reply source/packages/services/events-processor/src/api/targets/target.dao.spec.ts:66]: @hassankhokhar 
The purpose of this fix is to remove those redundant attributes. Those attributes will not be used by other logics - including events-alerts.
After removing those attributes the UT code needs to match the specification.
This above is the reason I removed those attributes.

File: source/packages/services/events-processor/src/api/targets/target.dao.ts (hunk 2 above)
Comment (bahavar) [reply source/packages/services/events-processor/src/api/targets/target.dao.ts:46]: @komushi , please see comments above

**Comment by boardthatpowder** [comment]
Closing this without merging due to inactivity. Please reopen and address outstanding review comments if it's still required.

**Comment by komushi** [comment]
After talking to @bahavar we know this PR change will not affect any existing customers - confirmed by @hassankhokhar and @bahavar 

We would like to merge it into the main branch.

**Comment by boardthatpowder** [comment]
@komushi just realized I had not published my review therefore my above comments related to this PR were not viewable.

**Comment by boardthatpowder** [comment]
Closing due to lack of activity.
